src/test/java/tests/MultiModelTest.java

```diff
@@ -23,7 +23,7 @@ public static void setup() {
         UI.status = new StatusUIStub();
     }
 
-    MultiModel multiModel = new MultiModel(new Preferences(true));
+    MultiModel multiModel = new MultiModel(mock(Preferences.class));
```

Comment:
I think it's not a strict guideline to use mocks as long as it's clear what's under test and the real classes can be used without too much additional logic. E.g. it's fine to use the original Preference here too. But sure, I will add some pointers regarding this to the docs.